lib/objectStorage.js

```diff
@@ -0,0 +1,78 @@
+const encryptor = require('./encryptor.js');
+const uuid = require('uuid');
+const axios = require('axios');
+const http = require('http');
+const https = require('https');
+
+class ObjectStorage {
+    constructor(settings) {
+        this.api = axios.create({
+            baseURL: `${settings.OBJECT_STORAGE_URI}/`,
+            httpAgent: new http.Agent({ keepAlive: true }),
+            httpsAgent: new https.Agent({ keepAlive: true }),
+            headers: { Authorization: `Bearer ${settings.OBJECT_STORAGE_TOKEN}` },
+            validateStatus: null
+        });
+    }
+
+    async requestRetry({ maxAttempts, delay, createReq, onRes }) {
```

Comment:
`createReq` and `onRes` seem not very clear names.
How about `request` (`retriableRequest`) and `callback`?

Reply:
`request` - ok
`callback` - bad name, too general. `onResponse` or leave as `onRes`.

Reply:
`onResponse` – better. I'd suggest using this.